streamduck-core/src/config.rs

```diff
@@ -530,6 +533,9 @@ pub struct DeviceConfig {
     pub layout: RawButtonPanel,
     pub images: HashMap<String, SDSerializedImage>,
     pub plugin_data: HashMap<String, Value>,
+    #[serde(skip)]
+    pub commit_time: Option<Instant>,
+    pub dirty_state: bool
```

Comment:
Probably add serde skip on dirty_state too, that attribute works only per field
```suggestion
    #[serde(skip)]
    pub dirty_state: bool
```